lucupy/sky/sun.py

```diff
@@ -156,15 +157,19 @@ def rise_and_set(location: EarthLocation,
                      midnight: Time,
                      set_alt: Angle,
                      rise_alt) -> Tuple[Time, Time, Time, Time]:
-        """
-        Compute rise and set times for this Sun, for the current
-        ``location`` and ``time`` of the night.
-
-        Returns
-        -------
-        `~astropy.time.Time`
-            The time of the event for the body in the
-            ``precision`` of this `Sun`.
+        """Compute rise and set times for this Sun. 
+        
+            For the current location and time of the night.
+
+        Args:
+            location (EarthLocation): Earth location
+            time (Time): time of the night.
+            midnight (Time): Midnight time.
+            set_alt (Angle): sunset altitude.
+            rise_alt (_type_): sunrise altitude.
```

Comment:
Not sure how we missed this. I think `rise_alt` should be an Astropy `Angle` like `set_alt`. We should throw this in the method signature and the doc.